patches/chrome-android-expectations-lint-suppressions.xml.patch

```diff
@@ -6,5 +6,5 @@ index 3c24f194a413e5abe3fab547e2eeb0b0ffcec7e7..b0147524d1683b05c24b1364106622a8
      <!-- Discussed in crbug.com/1069204, ignoring this class of errors since these are Q+ constants. -->
      <ignore regexp="Must be one of: LineBreaker.BREAK_STRATEGY_SIMPLE, LineBreaker.BREAK_STRATEGY_HIGH_QUALITY, LineBreaker.BREAK_STRATEGY_BALANCED"/>
    </issue>
-+  <issue id="DuplicateDefinition"><ignore regexp="chrome/android/java/res/values/brave_colors.xml"/></issue><issue id="StringFormatCount"><ignore regexp="sponsored_images_bottom_sheet_text"/><ignore regexp="notification_marketing"/><ignore regexp="renew_monthly_save"/><ignore regexp="wallet_swap_fees_notice"/><ignore regexp="processing_fee"/></issue><issue id="WrongConstant"><ignore regexp="ContentSettingsType.BRAVE_CONTENT_SETTINGS_TYPES_LIST"/></issue>
++  <issue id="DuplicateDefinition"><ignore regexp="chrome/android/java/res/values/brave_colors.xml"/></issue><issue id="StringFormatCount"><ignore regexp="sponsored_images_bottom_sheet_text"/><ignore regexp="notification_marketing"/><ignore regexp="renew_monthly_save"/><ignore regexp="wallet_swap_fees_notice"/><ignore regexp="processing_fee"/></issue><issue id="WrongConstant"><ignore regexp="ContentSettingsType.BRAVE_CONTENT_SETTINGS_TYPES_LIST"/></issue><issue id="UnusedResources"><ignore regexp="The resource `R.string.playlist_.*` appears to be unused"/></issue>
```

Comment:
> `The resource `R.string.playlist_.*` appears to be unused`

why are we adding this ignore? why not remove/fix unused resources?

Reply:
> > The resource `R.string.playlist_.*` appears to be unused
> 
> why are we adding this ignore? why not remove/fix unused resources?
We are using the strings from external module and to avoid duplicates we are basically overwriting the strings into brave project and ignoring the above warning. We need to have strings in brave project mainly for the translations.